bindings/python/aditofpython.cpp

```diff
@@ -247,6 +247,8 @@ PYBIND11_MODULE(aditofpython, m) {
              py::arg("value"))
         .def("getControl", &aditof::Camera::getControl, py::arg("control"),
              py::arg("value"))
+        .def("saveModuleCCB", &aditof::Camera::saveModuleCCB,
+             py::arg("filepath"))
```

Comment:
Recently these changes have reached master. So you can remove them to avoid duplication.